src/lib/operators/aggregate_hash.cpp

```diff
@@ -321,7 +343,12 @@ KeysPerChunk<AggregateKey> AggregateHash::_partition_by_groupby_keys() const {
                   if (position.is_null()) {
                     keys[chunk_offset] = 0;
                   } else {
-                    keys[chunk_offset] = int_to_uint(position.value()) + 1;
+                    const auto key = int_to_uint(position.value()) + 1;
```

Comment:
We only need the min/max for the single-groupby-column case, right? Use a constexpr to avoid tracking them for all other cases?

Reply:
You mean an `if constexpr`? That is already there, in line 342.

Reply:
Totally overlooked. 😳